ckanext/hierarchy/helpers.py

```diff
@@ -47,45 +46,40 @@ def group_tree_section(id_, type_='organization', include_parents=True,
         {'id': id_, 'type': type_, })
 
 
-def get_group_type_show(type_):
-    """ User with custom group types will be able to
-        use a "custom_type_show" action or the base action organization_show
-        for each new group type."""
-    custom_show_action = toolkit.config.get(
-        'ckanext.hierarchy.group_type_show_action.{}'.format(type_)
-    )
-    return custom_show_action or '{}_show'.format(type_)
+def _get_action_name(group_id):
+    model_obj = model.Group.get(group_id)
+    return "organization_show" if model_obj.is_organization else "group_show"
 
 
 def group_tree_parents(id_, type_='organization'):
```

Comment:
Drop the `_type` param in all functions if not used any more

Reply:
I forgot that. Thanks @amercader.
Fixed here https://github.com/ckan/ckanext-hierarchy/commit/c0c3a8d1e676b0b133ac3a5f1d270935e3c41edc